src/components/ebay-area-chart/component.js

```diff
@@ -0,0 +1,331 @@
+import Highcharts from 'highcharts';
+import accessibility from 'highcharts/modules/accessibility';
+import patternFill from 'highcharts/modules/pattern-fill';
+import {
+    backgroundColor,
+    gridColor,
+    labelsColor,
+    legendColor,
+    legendInactiveColor,
+    legendHoverColor,
+    tooltipBackgroundColor,
+    tooltipShadows,
+    setSeriesColors,
+} from '../../common/charts/colors';
+
+import { ebayLegend } from '../../common/charts/legend';
+
+if (typeof Highcharts === 'object') {
+    // enable highcharts accessibility with wrapper function
+    accessibility(Highcharts);
+    // patternFill highcharts wrapper function enables rendering patterns instead of just solid colors
+    patternFill(Highcharts);
+    // add custom legend wrapper function
+    ebayLegend(Highcharts);
+}
+const pointSize = 1.5;
+
+export default class {
+    onCreate() {
+        this.chartRef = null;
```

Comment:
Why is this needed?

Reply:
so that I can store a reference to the chart once it is initialized.  Later in the code I often call the redraw method, or look at data elements from the base chartRef.

Reply:
The question is, why set it at null initially? It's not doing anything. There's no checks if it's null etc.

Reply:
I thought it was nice to see a reference to a variable that is called later down, and the linter does let me create a variable with no assignment initially.  I'll just remove these lines then.  Was trying to make reading this file easier